docs/changelogs/CHANGELOG-2.23.md

```diff
@@ -11,6 +11,20 @@
 - [v2.23.8](#v2238)
 - [v2.23.9](#v2239)
 - [v2.23.10](#v22310)
+- [v2.23.11](#v22311)
+
+## [2.23.11](https://github.com/kubermatic/kubermatic/releases/tag/2.23.11)
+
+### Bugfixes
+
+- **ACTION REQUIRED:** If you use `velero.restic.deploy: true`, you will see new daemonset `node-agent` running in `velero` namespace. You might need to remove existing daemonset named `restic` manually ([#12998](https://github.com/kubermatic/kubermatic/pull/12998))
```

Comment:
Can we mention that this is related to the Velero Helm chart? With the soon-to-be-added Velero integration it could be confusing to know what this changelog item is referring to.